org.electronjs.Electron2.BaseApp.yml

```
@@ -85,3 +84,25 @@ modules:
         x-checker-data:
           type: git
           tag-pattern: ^v([\d.]+)$
+
+  - name: setup-separate-locales
+    buildsystem: simple
+    build-commands:
+      - install -m755 setup-separate-locales /app/bin/setup-separate-locales
+    sources:
+      - type: script
+        dest-filename: setup-separate-locales
+        commands:
+          - |
+            APP_LOCALES_DIR=$1
```

I think it would be better if the script would accept the app root dir as argument, not the locales dirs. The `/locales` subdir is always the same, so it doesn't make much sense to add to every script invocation.
```suggestion
            APP_LOCALES_DIR=$1/locales
```